client/signup/steps/domains/index.jsx

```diff
@@ -300,6 +300,13 @@ class RenderDomainsStepComponent extends Component {
 
 		await this.props.saveSignupStep( stepData );
 
+		if ( this.props.shouldUseDomainSearchV2 && suggestion?.isSubDomainSuggestion ) {
```

Comment:
Should we also check whether it's a free domain? I'm not sure of the difference between `is_free` and `isSubDomainSuggestion`. To me, they look like the same thing, but I know nothing 😅

@hambai, thoughts?

Reply:
The `isSubDomainSuggestion` check is enough